app/presenters/user_csv_presenter.rb

```diff
@@ -16,7 +16,7 @@ def self.to_csv(
     inherited: false, deleted: false, project_id: nil, user_id: nil, datetime: (Time.now.strftime "%Y%m%d_%H%M")
   )
     inherited = true if project_id || user_id
-    users = (deleted || user_id) ? User.unscoped : User
+    users = (deleted || user_id ? User.unscoped : User)
```

Comment:
I like these since it makes it obvious that we assign something / that the right part could be extracted into a method